LayoutTests/fast/scrolling/ios/ui-side-compositing-check.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <script src="../../../resources/testharness.js"></script>
```

Comment:
Don't use test harness in non-WPT tests. Better to use `js-test-pre.js`,` js-test-post.js`